app/views/trips/show.html.erb

```diff
@@ -0,0 +1,28 @@
+<%= render partial: 'layouts/style' %>
+<%= render partial: 'layouts/nav' %>
+
+<section class="cool-list heebo-list margin-left show">
+<h2> Trip # <%= @trip.id %> </h2>
+  <ul>
+    <li>
+      Date: <%= @trip.date %>
+    </li>
+    <li>
+      Driver: <%= link_to @trip.driver.name, driver_path(@trip.driver.id) %>
+    </li>
+    <li>
+      Rating: <%= @trip.rating %>
+    </li>
+    <li>
+      Cost: $<%= '%.2f'%(@trip.cost/100) %>
```

Comment:
You'll learn more about view helpers later - but this would be a great place to use one so that you don't have to copy/paste the way to format $ every time you need it in a view